packages/account-v2/src/containers/Onfido/OnfidoView.classnames.ts

```diff
@@ -0,0 +1,9 @@
+import { cva } from 'class-variance-authority';
+
+export const OnfidoCustomClass = cva('onfido-custom-container', {
```

Comment:
Lets restrict **Title** case for components. To identify variants may be we can try suffixing variable name with `variant` at the end?

Reply:
Updated.